src/components/Footer/Footer.tsx

```diff
@@ -38,6 +38,50 @@ const Footer = () => {
 
           <nav className="grid-col-auto usa-footer__nav">
             <div className="grid-row grid-gap">
+              <div className="grid-col">
+                <section className="usa-footer__primary-content usa-footer__primary-content--collapsible">
+                  <ul className="usa-list usa-list--unstyled">
+                    <li className="usa-footer__secondary-link">
+                      <LinkTo href="/" rel="noreferrer noopener">
+                        My Space
+                      </LinkTo>
+                    </li>
+                    <li className="usa-footer__secondary-link">
+                      <LinkTo
+                        href="/sites-and-applications"
+                        rel="noreferrer noopener">
+                        All Sites & Applications
+                      </LinkTo>
+                    </li>
+                    <li className="usa-footer__secondary-link">
+                      <LinkTo
+                        href="/ussf-documentation"
+                        rel="noreferrer noopener">
+                        USSF Documentation
+                      </LinkTo>
+                    </li>
+                    <li className="usa-footer__secondary-link">
+                      <LinkTo href="/about-us" rel="noreferrer noopener">
+                        About the USSF
+                      </LinkTo>
+                    </li>
+                    <li className="usa-footer__secondary-link">
+                      <LinkTo
+                        href="/about-us/orbit-blog"
+                        rel="noreferrer noopener">
+                        Orbit Blog
+                      </LinkTo>
+                    </li>
+                    <li className="usa-footer__secondary-link">
+                      <LinkTo
+                        href="/news-announcements"
+                        rel="noreferrer noopener">
+                        News & Announcements
+                      </LinkTo>
+                    </li>
+                  </ul>
+                </section>
+              </div>
```

Comment:
I took a look at why the e2e tests failed, and it looks like it's because you added this section. Some of the e2e tests look for the text `All Sites & Applications` to click, but now that this is here, that search for text is resolving to 2 elements, causing the test to fail. That might also be the case for others in this section. You'll need to create a new branch in the `ussf-portal` repo and update the tests there. If you'd like I can help you with it, or take that on completely if you've got too much on your plate.